include/openssl/store.h

```diff
@@ -249,8 +254,6 @@ int OSSL_STORE_LOADER_set_open(OSSL_STORE_LOADER *loader,
 typedef OSSL_STORE_LOADER_CTX *(*OSSL_STORE_attach_fn)(const OSSL_STORE_LOADER
                                                        *loader,
                                                        BIO *bio,
-                                                       OPENSSL_CTX *libctx,
-                                                       const char *propq,
```

Comment:
Why remove these?

Reply:
@levitte
Inside loader_file.c we have
```
file_load(OSSL_STORE_LOADER_CTX *ctx, const UI_METHOD *ui_method, void *ui_data);
```
And this function needs the libctx/propq.

So are you happy having a libctx/propq stored in ossl_store_loader_ctx_st,
that gets copied during the file_open()?

Reply:
... and file_attach() also

Reply:
> So are you happy having a libctx/propq stored in ossl_store_loader_ctx_st, that gets copied during the file_open()?

Yes, that's exactly where I think we should go.

> (I thought I remember doing this before and then you may have complained that there was a libctx in there..)

It's not *specifically* about this.  I believe it's a mistake not to treat structures like RSA as free objects, not attached to a library context...  that was a losing battle though, so here we are, and we do need to pass libctx and propq in ways I didn't expect at the time.  This is one of them.  In other word, the protest I made at the time isn't valid with the code we currently have.